pkgs/development/python-modules/astropy/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchPypi
+, stdenv 
+, python3Packages
+, buildPythonPackage
+, numpy }:
+
+buildPythonPackage rec {
+    pname = "astropy";
+    version = "1.3.3";
+
+    name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ed093e033fcbee5a3ec122420c3376f8a80f74663214560727d3defe82170a99";
+  };
+  doCheck = false;
```

Comment:
That was an oversight. Astropy has 9000 tests so feels like you are waiting for the heat death of the universe when checking both python builds. I've re-enabled them. I'll submit the PR when they are done.